src/components/account/AccountCreation.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div v-if="errors.length">
+    <b>Please correct the following error(s):</b>
+    <ul>
+      <li v-for="error in errors" :key="error">{{ error }}</li>
+    </ul>
+  </div>
+  <div class="field">
+    <p class="control has-icons-left has-icons-right">
+      <input class="input" type="email" placeholder="Email" v-model="mail">
+      <span class="icon is-small is-left">
+      <i class="fas fa-envelope"></i>
+    </span>
+      <span class="icon is-small is-right">
+      <i class="fas fa-check"></i>
+    </span>
+    </p>
+  </div>
+  <div class="field">
+    <p class="control has-icons-left">
+      <input class="input" type="password" placeholder="Password" v-model="password">
+      <span class="icon is-small is-left">
+      <i class="fas fa-lock"></i>
+    </span>
+    </p>
+    <p class="control has-icons-left">
+      <input class="input" type="password" placeholder="Password" v-model="passwordVerification">
+      <span class="icon is-small is-left">
+      <i class="fas fa-lock"></i>
+    </span>
+    </p>
+  </div>
```

Comment:
Ajouter un champ pour le nom du compte (ex: Dupond), car il est en obligatoire dans la DataClass de Account.

Reply:
Le nom n'a pas a être obligatoire
related to https://github.com/Prakkkmak/animaliste-back/issues/12